randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -844,8 +844,9 @@ Extra - room_id: [26]
   > Door to Security Access (Upper)
       Space Jump
   > Door to Crumble City
-      # TODO: Look into requirements further and possibly add SWJ trick. The requirement of space jump makes the advanced tricks in crumble city invalid in non-entrance rando.
-      Screw Attack and Space Jump
+      All of the following:
+          Screw Attack
+          Space Jump or Wall Jump (Beginner)
```

Comment:
is tihs supposd to be a screw single walljump? lock it behind a template please.

Reply:
This is a template called "Can Single Wall Jump" already, but looking at the header file, it looks like what is listed under it is "Impossible."

Reply:
This is intended, because there is no final conclusion yet on how SJW behaves. Having them behind a template means we can then later fix them all easily.

Reply:
Perfect! Good to know! Then yeah, this should likely read

ALL OF:
Screw Attack
      OR
      Space Jump or Can Single Wall Jump

Reply:
After Jagger found a way to get to the screw blocks with speed booster, we should update this to:

ALL OF:
Screw Attack
       ANY OF:
       Space Jump or Speed Booster or Can Single Wall Jump

Reply:
the speedbooster needs a shinespark, a level 1 door, no entrance rando and no door lock rando and probably other things i have forgotten